packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -16,6 +16,10 @@
   "force_source_status_text":
     "SFDX: View All Changes (Local and in Default Scratch Org)",
   "force_apex_test_run_text": "SFDX: Invoke Apex Tests...",
+  "force_apex_test_run_codeAction_all_tests_text":
```

Comment:
@cwallsfdc - These two don't seem to be used anywhere anymore after your latest commit. Should we remove them?